tests/improver-wxcode/01-help.bats

```diff
@@ -41,21 +41,21 @@ Calculate gridded weather symbol codes.
 This plugin requires a specific set of input diagnostics, where data
 may be in any units to which the thresholds given below can
 be converted:
- - probability_of_rainfall_rate; thresholds: above 0.03 (mm hr-1), above 0.1 (mm hr-1), above 1.0 (mm hr-1)
- - probability_of_lwe_snowfall_rate; thresholds: above 0.1 (mm hr-1), above 1.0 (mm hr-1)
- - probability_of_cloud_area_fraction; thresholds: above 0.1875 (1), above 0.8125 (1)
- - probability_of_visibility_in_air; thresholds: below 1000.0 (m), below 5000.0 (m)
- - probability_of_cloud_area_fraction_assuming_only_consider_surface_to_1000_feet_asl; thresholds: above 0.85 (1)
- - probability_of_rainfall_rate_in_vicinity; thresholds: above 0.1 (mm hr-1), above 1.0 (mm hr-1)
- - probability_of_lwe_snowfall_rate_in_vicinity; thresholds: above 0.1 (mm hr-1), above 1.0 (mm hr-1)
+ - probability_of_rainfall_rate_above_threshold; thresholds: above 0.03 (mm hr-1), above 0.1 (mm hr-1), above 1.0 (mm hr-1)
+ - probability_of_lwe_snowfall_rate_above_threshold; thresholds: above 0.1 (mm hr-1), above 1.0 (mm hr-1)
+ - probability_of_cloud_area_fraction_above_threshold; thresholds: above 0.1875 (1), above 0.8125 (1)
+ - probability_of_visibility_in_air_below_threshold; thresholds: below 1000.0 (m), below 5000.0 (m)
+ - probability_of_cloud_area_fraction_assuming_only_consider_surface_to_1000_feet_asl_above_threshold; thresholds: above 0.85 (1)
+ - probability_of_rainfall_rate_in_vicinity_above_threshold; thresholds: above 0.1 (mm hr-1), above 1.0 (mm hr-1)
+ - probability_of_lwe_snowfall_rate_in_vicinity_above_threshold; thresholds: above 0.1 (mm hr-1), above 1.0 (mm hr-1)
```

Comment:
We could now get rid of the "above" and "below" that precede each value above. The name of the diagnostic makes it clear in which sense we are interested relative to these thresholds, e.g.
```
probability_of_visibility_in_air_below_threshold; thresholds: 1000.0 (m), 5000.0 (m)
```

Reply:
Good point, updated